### PR TITLE
rebranding

### PR DESCRIPTION
**Pull request by moldover**
Fix for #584 rebranding.

### CODE REVIEW

File: src/main/java/network/brightspots/rcv/CastVoteRecord.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Universal RCV Tabulator
+ * RCTab
  * Copyright (c) 2017-2020 Bright Spots Developers.
```
Comment (HEdingfield) [inline src/main/java/network/brightspots/rcv/CastVoteRecord.java:3]: Can you please update the copyright date to 2022 for all these files while you're at it?

**Review by HEdingfield — COMMENTED** [review]
I found two other locations that should probably be updated too:
* [BrightSpotsCopyrightProfile.xml](https://github.com/BrightSpots/rcv/blob/26ae62fdd9402d9fd0d3f435771fb90553843dd1/.idea/copyright/BrightSpotsCopyrightProfile.xml#L3)
* [Clear Ballot 2.1 RCV Format Specification.rtf](https://github.com/BrightSpots/rcv/blob/26ae62fdd9402d9fd0d3f435771fb90553843dd1/reference/clear_ballot/RCV/Clear%20Ballot%202.1%20RCV%20Format%20Specification.rtf#L74)

**Review by rrojas350 — APPROVED** [review]
Main.java has a few lines that refer to the "RCV tabulator" but they're comments, on line 29 and 38.

It's not a major change or show stopper but that's all I found besides the ones Hylton pointed out.

File: src/main/java/network/brightspots/rcv/CastVoteRecord.java (hunk 1 above)
Comment (moldover) [reply src/main/java/network/brightspots/rcv/CastVoteRecord.java:3]: I think it's better to do this along with the license updates in a different PR.

Comment (moldover) [reply src/main/java/network/brightspots/rcv/CastVoteRecord.java:3]: I fixed the copyright template, and updated the comments in Main.java.  The .rtf here is a document we received from Clear Ballot, so not something we want to touch.

**Pull request merged**